<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerEmulatorOptionsWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerEmulatorOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="emulatorLabel">
         <property name="text">
          <string>Virtua&amp;l Machine:</string>
         </property>
         <property name="buddy">
          <cstring>emulatorComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="emulatorComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Mer::Internal::MerEmulatorDetailsWidget" name="emulatorDetailsWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>&amp;Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startVirtualMachineButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&amp;Start Virtual Machine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopVirtualMachineButton">
       <property name="text">
        <string>Sto&amp;p Virtual Machine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="regenerateSshKeysButton">
       <property name="text">
        <string>Regenerate SSH Keys</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="emulatorModeButton">
       <property name="text">
        <string>Emulator Mode...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="factoryResetButton">
       <property name="text">
        <string>Factory Reset...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Mer::Internal::MerEmulatorDetailsWidget</class>
   <extends>QWidget</extends>
   <header>mer/meremulatordetailswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
